index.js

```diff
@@ -30,7 +30,8 @@ if (cmd === "help" || args.some((a) => a === "-h" || a === "--help")) {
 }
 
 const isDev = cmd === "dev";
-const port = 5000;
+const portOptionIndex = args.findIndex((a) => a === "-p" || a === "--port")
+const port = (~portOptionIndex && args[portOptionIndex + 1]) || 5000
```

Comment:
we should remove this option and value from `args` after line 34 because, args are forwarded to posthtml-cli:

```js
let port = 5000;
if (~portOptionIndex && args[portOptionIndex + 1]) {
  port = args[portOptionIndex + 1];
  args.splice(portOptionIndex, 2);
}
```